easyconfigs/b/Barracuda_Virtual_Reactor/Barracuda_Virtual_Reactor-22.1.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CmdCp'
+
+name = 'Barracuda_Virtual_Reactor'
+version = '22.1.1'
+
+homepage = 'https://cpfd-software.com'
+description = "The world leader in fluidized bed reactor simulation software for CAE/CFD."
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+# The source URL will likely need tweaking, depending on when the version was released
+source_urls = ['https://cpfd-software.com/wp-content/uploads/2022/02/']
+sources = ['%(namelower)s-%(version)s-Linux.tar.gz']
+checksums = ['8537b1b7d001808e6610e31220ef826de91c4ea1a183fe8f4967a97a8af1360e']
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+dependencies = [
+    ('Python', '3.10.4'),
+    ('X11', '20220504'),
+]
+
+skipsteps = ['install']  # the install step blows-away the directory created by the above command
+files_to_copy = []
+cmds_map = [
+    ('.*', 'rm -rf %(installdir)s && ./%(namelower)s-%(version)s-Linux.run install --accept-licenses --confirm-command --auto-answer installationErrorWithCancel=Ignore --default-answer --root %(installdir)s'),
```

Comment:
I'm surprised flake8 didn't complain! I thought there was a max length for lines?